scanpy/ann_data.py

```diff
@@ -91,21 +92,20 @@ def _check_dimensions(data, smp, var):
                          .format(nr_var, var.shape[0]))
 
 class AnnData(IndexMixin):
-    def __init__(self, X, smp=None, var=None, vis=None, **meta):
-        u"""
-        Annotated Matrix
+    def __init__(self, ddata=None, X=None, smp=None, var=None, vis=None, **meta):
+        """
+        Annotated Data
 
-        Represents a sample × variable matrix (e.g. cell × gene) with the
-        possibility to store an arbitrary number of annotations for both
-        samples and variables.
+        Stores a data matrix X of dimensions nr_samples x nr_variables,
```

Comment:
Because of compatibility with Python 2, which threw an error. I think that could have been fixed by specifying the encoding. But then, everything should be done the same way. I can restore it when doing some final polishing on the docstrings.